RELEASING.md

```diff
@@ -4,35 +4,35 @@ The release process for SNAPSHOT versions is handled by the `after_success.sh` s
 
 ### To push a release version
 
-1. Make sure the team agrees that it is time to release. Verify that all unit and integration tests for the last commit have passed.
+1. Make sure the team agrees that it is time to release. 
 
-2. Run `utilities/update_versions.sh` from the repository's base directory.
-This script takes optional arguments denoting the new versions for each qualifier (alpha, beta and/or GA). By default, if the current version is X.Y.Z-SNAPSHOT, the script will update the version in all the pom.xml and other relevant files to X.Y.Z. Please refer to the documentation in `utilities/update_versions.sh` for more details. Commit this version locally:
+2. Look over all of the commits since the last release and make sure there are no breaking changes on the public surface. If there are any breaking changes, create and merge a new PR to revert the surface back.
```

Comment:
The review should only check for changes that are observable in javadoc. No implementation changes need to be checked, no README changes need to be checked, no snippet changes need to be checked; it is just a quick scan for changes that will break compilation of users. I will clarify this.